service.py

```diff
@@ -52,14 +54,5 @@ def parseRecord(encodedRec):
         logger.debug(b64Err)
         return False
 
-    status = record['status']
-    stage = record['stage']
-    if status != 200:
-        logger.warning('Bad Record Found! Alert the Authorities')
-        return False
-    elif stage != 'oclc':
-        logger.info('This record is not for this stage, return for further processing')
-        return False
-
     result = enhanceRecord(record)
```

Comment:
I think I would put this and the return in the `try`, take `return False` out of the `excepts` and make _it_ the last-ditch return

Also `enhanceRecord()` does a lot of error checking and returning `False`. It might read more cleanly if `enhanceRecord()` just threw errors which were caught here, in one place, and logged.

Reply:
Both good ideas I think for making the control flow simpler and the methods easier to read.